openedx/core/djangoapps/user_authn/views/tests/test_logistration.py

```diff
@@ -87,11 +84,6 @@ def test_logistration_mfe_redirects(self, url_name, path):
         response = self.client.get(reverse(url_name))
         self.assertContains(response, 'Sign in or Register')
 
-        # Test with setting enabled and waffle flag active, redirects to microfrontend
-        with override_waffle_flag(REDIRECT_TO_AUTHN_MICROFRONTEND, active=True):
-            response = self.client.get(reverse(url_name))
-            self.assertRedirects(response, settings.AUTHN_MICROFRONTEND_URL + path, fetch_redirect_response=False)
```

Comment:
Why we are removing these two lines, these actually testing the redirection.

Reply:
fixed it. thanks